### PR TITLE
Dockerignore

### LINKED ISSUE #77: Use a .dockerignore file to reduce build context
Currently the whole project is sent to docker as the context when you do docker build . -t my-fantastic-image. This is currently tens of GB. Use dockerignore to only send the two files needed - Dockerfile and DESCRIPTION. No other files are needed at build time.

### PR DESCRIPTION
**Pull request by joeHickson**
fixes #77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,7 @@ To build the docker image run (from the `covid` directory):
 docker build . -t covid
 ```
 
-Note that this command fails on a vm with only 4gb of ram. Minimum spec has not yet been established but is higher than this!
-For a faster build copy Dockerfile and DESCRIPTION to an empty directory and run the above command there.
+Note that the build command fails on a vm with only 4gb of ram. Minimum spec has not yet been established but is higher than this!
```
Comment (seabbs) [inline README.md:50]: With only the Dockerfile and DESCRIPTION in scope is this still the case?

**Comment by seabbs** [comment]
Looks good - happy for it to be merge in as and when. Just checking that the build still fails on 4gb servers?

**Comment by joeHickson** [comment]
I think so - it was the compilation of the modules that used up the ram,
not the context

**Comment by seabbs** [comment]
ah that is a shame.

**Pull request merged**